src/validate.py

```diff
@@ -144,3 +126,53 @@ def parse_accept_language(header, supported_langs):
 
     # If all else fails, default the language.
     return DEFAULT_LANGUAGE.lang_code
+
+def get_json_files_from_dir(path):
+
+    files_found = []
+    for root, directories, files in os.walk(path):
+        for file in files:
+            if '.json' in file:
+                files_found.append(os.path.join(root, file))
+
+    return files_found
+
+def get_entries_from_json(path, p_key, s_key):
+
+    with open(path) as json_file:
+        data = json.load(json_file)
+
+    entries = []
+
+    if p_key in data:
+        for values in data.get(p_key):
+             entries.append(values.get(s_key))
+
+    return entries
+
+def get_chapters_for_year(file):
+
+    chapters = []
+
+    data = get_entries_from_json(file,'outline','chapters')
+    for list in data:
+        for entry in list:
+            chapters.append(entry.get('title').lower().replace(" ", "-").replace("/",""))
+
+    return chapters
+
+def get_parts_for_year(file):
+
+    parts = get_entries_from_json(file,'outline','part')
+
+    return parts
+
+json_files_present = get_json_files_from_dir(CONFIG_DIR)
+
+
+#TO DO - Lazy load previous year chapters
+
+for file in json_files_present:
+    if DEFAULT_YEAR in file:
```

Comment:
Don't understand this. So it will ONLY load the default year JSON file? That's not a default then (which implies a value to fall back to if nothing is specified/found) - but a hardcoded value. Which, given my other concerns seems like a problem.

Reply:
right.. so I wanted to confirm that.  will be having multiple json files for each year ?

Reply:
I think it'll be one per year.

Reply:
It will default to the latest year among the list of json files.